packages/data-lake-api/.env.example

```diff
@@ -0,0 +1,7 @@
+DATA_LAKE_DB_NAME=
+DATA_LAKE_DB_PASSWORD=
+DATA_LAKE_DB_URL=
+DATA_LAKE_DB_USER=
+
+DB_PG_USER=
+DB_PG_PASSWORD=
```

Comment:
Need the PG_USER to be able to create the `data_lake_test` database.

Reply:
Yep test only. I'll keep the name as is as it's consistent with other packages, but I'll add a comment like you suggest 👍